### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by liamsi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,8 +64,8 @@ According to SemVer, anything in the public API can change at any time before ve
 
 - [LazyLedger Ethereum research post](https://ethresear.ch/t/a-data-availability-blockchain-with-sub-linear-full-block-validation/5503)
 - [LazyLedger academic paper](https://arxiv.org/abs/1905.09274)
-- [Blog](https://medium.com/lazyledger)
-- [Project web site](https://lazyledger.io/)
+- [Blog](https://blog.celestia.org)
+- [Project web site](https://celestia.org)
 - [Academic LazyLedger prototype](https://github.com/LazyLedger/lazyledger-prototype)
 - [Follow LazyLedger on Twitter](https://twitter.com/lazyledger_io)
```
Comment (liamsi) [inline README.md:70]:
```suggestion
- [Follow Celestia on Twitter](https://twitter.com/CelestiaOrg)
```

**Comment by adlerjohn** [comment]
The changes look fine, but still missing:

1. Delete old lazyledger logo file
2. Change Telegram, and forum (specs can stay as-is for now)

**Review by evan-forbes — APPROVED** [review]
LGTM! :+1:

**Review by adlerjohn — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ See the [install instructions](/docs/introduction/install.md).
 Before contributing to the project, please take a look at the [contributing guidelines](CONTRIBUTING.md)
 and the [style guide](STYLE_GUIDE.md).
 
-Join the community at [Telegram](https://t.me/lazyledgerchat) or jump onto the [Forum](https://talk.lazyledger.io/)
+Join the community at [Telegram](https://t.me/CelestiaCommunity) or jump onto the [Forum](https://talk.lazyledger.io/)
```
Comment (musalbas) [inline README.md:48]: talk.lazyledger.io -> forum.celestia.org

**Review by musalbas — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (liamsi) [inline README.md:48]:
```suggestion
Join the community at [Telegram](https://t.me/CelestiaCommunity) or jump onto the [Forum](https://forum.celestia.org/)
```

**Review by musalbas — APPROVED** [review]
LGMT but we should change the GitHub URL to the prototype and spec, when it's final.

**Pull request merged**